kitsune/sumo/jinja2/base.html

```diff
@@ -327,6 +327,8 @@ <h5 class="mzp-c-footer-heading">{{ _('Firefox Accounts') }}</h5>
   <script src="{{ static('jsi18n/{lang}/djangojs.js'|f(lang=request.LANGUAGE_CODE|lower)) }}"></script>
 {% endif %}
 
+<script src="{{ url("wafflejs") }}"></script>
```

Comment:
How about using waffle to control whether we are going to load this?

Reply:
This is a nit feel free to ignore. This will be cached but for the use case that we have it shouldn't matter.

Reply:
I'll follow up with some ideas I had to avoid the caching problem in another PR